aggregator_core/src/datastore.rs

```diff
@@ -71,13 +71,19 @@ pub mod test_util;
 ///
 /// [1]: https://docs.rs/rstest_reuse/latest/rstest_reuse/
 macro_rules! supported_schema_versions {
-    ($( $i:literal ),*) => {
-        const SUPPORTED_SCHEMA_VERSIONS: &[i64] = &[$($i),*];
+    ( $i_latest:literal $(,)? $( $i:literal ),* ) => {
+        const SUPPORTED_SCHEMA_VERSIONS: &[i64] = &[$i_latest, $($i),*];
 
         #[cfg(test)]
         #[rstest_reuse::template]
         #[rstest::rstest]
-        $(#[case(ephemeral_datastore_max_schema_version($i))])*
+        // Test the latest supported schema version.
+        #[case(ephemeral_datastore_schema_version($i_latest))]
```

Comment:
I would have liked the cases to have neater titles than `case_1`, `case_2`, etc.

In rstest, that's done using this technique:
```
#[case::foobar(args)]
```
Where `foobar` is a valid Rust `ident`(ifier).

Sadly there doesn't seem to be a _stable_ way to construct idents out of macro arguments, something like the `##` operator in the C preprocessor. In Rust-land it's the experimental [`concat_idents!()`](https://doc.rust-lang.org/std/macro.concat_idents.html) macro.